src/misc/lv_anim.c

```diff
@@ -45,9 +45,9 @@ static int32_t lv_anim_path_cubic_bezier(const lv_anim_t * a, int32_t x1,
  *      MACROS
  **********************/
 #if LV_LOG_TRACE_ANIM
-    #define TRACE_ANIM(...) LV_LOG_TRACE(__VA_ARGS__)
+    #define LV_TRACE_ANIM(...) LV_LOG_TRACE(__VA_ARGS__)
 #else
-    #define TRACE_ANIM(...)
+    #define LV_TRACE_ANIM(...)
```

Comment:
It's not really a global policy but rather my personal preference to add the `lv_` prefix only to global functions/enums/types to differentiate them from static and local declarations. 

So do you thin kit's easier to read the code this way?

Reply:
The aim is to have same naming convention in C files where we use tracing macros.
I saw that the tracing macro name differs across files, so I thought to follow the naming convention: LV_TRACE_xxxx.

Reply:
Could you please advise me the name convention to this tracing macro?
I will amend the PR with the new names.

Reply:
Oh, I haven't noticed that it's used inconsistently. Let's use your suggestion :+1: